### PR TITLE
Beretta action: Electric boogaloo

### PR DESCRIPTION
**Pull request by Irradiation**
First part of a three PRs project to make a new traitor bundle centered around akimbo pistols.
This PR adds the Beretta 92FS to the game, holding 15 9mm rounds per magazine. In addition, the Beretta 92FS is also added to the Summon Guns list for Wizards.

:cl:
	*rscadd: Added the Beretta 92 FS handgun. Kong whiskey not included.

### CODE REVIEW

**Comment by Irradiation** [comment]
https://youtu.be/cdIha5Dz07U?t=2m38s

File: code/modules/projectiles/guns/projectile/pistol.dm [hunk 1]
```diff
@@ -184,3 +184,21 @@
...
+	
+/obj/item/weapon/gun/projectile/beretta
+	name = "\improper Beretta 92FS"
+	desc = "The classic wonder nine and favorite of the undercover cop. Kong whiskey not included."
+	icon = 'icons/obj/beretta.dmi'
+	icon_state = "beretta"
+	max_shells = 15
+	caliber = list("9mm" = 1)
+	silenced = 0
+	origin_tech = Tc_COMBAT + "=2;" + Tc_MATERIALS + "=2" + Tc_SYNDICATE + "=2"
```
Comment (DamianX) [inline code/modules/projectiles/guns/projectile/pistol.dm]: Aren't you missing a `;` here?  `Tc_MATERIALS + "=2"`
datums for these when?

Comment (Irradiation) [reply code/modules/projectiles/guns/projectile/pistol.dm]: Fixed it. My copy-pasta skills are getting worse.

**Comment by Irradiation** [comment]
https://cdn.discordapp.com/attachments/239823497365422091/449729885200449546/Capture.PNG For those that don't like looking into .dmi files.

**Comment by SonixApache** [comment]
@Irradiation believe it or not, that's what IconDiffBot does. Click the little > icon.

**Comment by Irradiation** [comment]
Sure, but there's a difference between a static icon and seeing it "live".

**Comment by Exxion** [comment]
But that's still a static icon

**Comment by Irradiation** [comment]
IT LOOKS PRETTIER IN THE PICTURE, OKAY.

**Pull request merged**